### DISCUSSION TITLE
Activate "Constrain proportions" by default for images

### DISCUSSION THREAD

**Discussion by girafic** [uxbox/uxbox#541] [Ideas]
"Constrain proportions" should be activated by default for new images.

<img width="60" alt="grafik" src="https://user-images.githubusercontent.com/1187276/106743296-7b61ef80-661e-11eb-8dd5-f2f4a2c35f07.png">

**Comment by myfunnyandy** [comment]
Nice one! We are definitely adding it as an enhancement.

**Comment by girafic** [comment]
Discussion can be closed or deleted. Feature is already implemented.
